### PR TITLE
Add fluentd arm image for nerdctl

### PR DESCRIPTION
**Pull request by yankay**
What:
Add fluentd arm image for nerdctl

Why:
This image was used in nerdctl in https://github.com/containerd/nerdctl/pull/3045

### CODE REVIEW

**Comment by yankay** [comment]
HI @ktock, would you please help to review it? :-)

**Comment by AkihiroSuda** [comment]
ping @ktock

File: list_of_images_to_optimize.py [hunk 1]
```diff
@@ -59,5 +59,8 @@
     Image("python:3.10-slim", ["python", "-c", "print('hello world')"]),
     Image("python:3.11.0rc2", ["python", "-c", "print('hello world')"]),
     Image("pypy:3.9", ["pypy3", "-c", "print('hello world')"]),
-    Image("node:17.8.0", ["node", "-e", 'console.log("hello")'])
+    Image("node:17.8.0", ["node", "-e", 'console.log("hello")']),
+
+    Image("fluent/fluentd:v1.14-1", ["tini" "--" "/bin/entrypoint.sh"]),
+    Image("fluent/fluentd:v1.14-arm64-debian", ["tini" "--" "/bin/entrypoint.sh"])
```
Comment (ktock) [inline list_of_images_to_optimize.py]: delimiters are needed

Comment (yankay) [reply list_of_images_to_optimize.py]: Thanks @ktock 
It has been changed

**Review by ktock — APPROVED** [review]
Thanks

**Comment by yankay** [comment]
The job failed with 403 Forbidden: https://github.com/stargz-containers/image-ci/actions/runs/10202170474/job/28225691301#step:6:334

It's strange 🤔

**Comment by yankay** [comment]
> The job failed with 403 Forbidden: https://github.com/stargz-containers/image-ci/actions/runs/10202170474/job/28225691301#step:6:334
> 
> It's strange 🤔

HI @ktock 

The image push failed by 403 Forbidden, It may fixed by:

image

Ref: https://github.com/docker/build-push-action/issues/687#issuecomment-1238980158

**Comment by ktock** [comment]
@yankay Sorry for slow reply. Fixed.  https://github.com/stargz-containers/image-ci/pkgs/container/fluent%2Ffluentd

**Pull request merged**